huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -551,6 +565,10 @@ func resourceRdsInstanceRead(ctx context.Context, d *schema.ResourceData, meta i
 		d.Set("fixed_ip", privateIps[0])
 	}
 
+	maintainWindow := strings.Split(instance.MaintenanceWindow, "-")
```

Comment:
how can we make sure than the length of `maintainWindow` is greater than 2.

Reply:
done, but in reality, the result will definitely contain a start time and an end time, and seperated by -